src/onegov/form/fields.py

```diff
@@ -460,6 +462,11 @@ class TextAreaFieldWithTextModules(TextAreaField):
     widget = TextAreaWithTextModules()
 
 
+class VideoURLField(URLField):
+
+    widget = VideoURLFieldRenderer()
```

Comment:
```suggestion
    pass
```
The original widget should do the job just fine, if you need to adjust the display, then you should create a widget that derives from `URLInput` and applies any necessary changes.